packages/react-components/src/atoms/__tests__/NavigationLink.test.tsx

```diff
@@ -74,4 +74,15 @@ describe('with a router', () => {
     );
     expect(fireEvent.click(getByRole('link'))).toBe(false);
   });
+
+  it('triggers a full page navigation on click of an external link', () => {
+    const { getByRole } = render(
+      <StaticRouter>
+        <NavigationLink href="http://test.com/" icon={<svg />}>
```

Comment:
FYI `example.com` (and some others) are the [official example domains](https://en.wikipedia.org/wiki/Example.com). Let's prefer those over domains that somebody actually uses (already are in most places).
```suggestion
        <NavigationLink href="http://example.com/" icon={<svg />}>
```